extensions/kafka-streams/runtime/src/main/java/io/quarkus/kafka/streams/runtime/KafkaStreamsRuntimeConfig.java

```diff
@@ -0,0 +1,42 @@
+package io.quarkus.kafka.streams.runtime;
+
+import java.util.Optional;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+@ConfigRoot(name = "kafka-streams", phase = ConfigPhase.RUN_TIME)
+public class KafkaStreamsRuntimeConfig {
+
+    /**
+     * A unique identifier for this Kafka Streams application.
+     */
+    @ConfigItem
+    public String applicationId;
+
+    /**
+     * A comma-separated list of host:port pairs identifying the Kafka bootstrap server(s)
+     */
+    @ConfigItem(defaultValue = "localhost:9012")
+    public String bootstrapServers;
+
+    /**
+     * A unique identifier of this application instance, typically in the form host:port.
+     */
+    @ConfigItem
+    public Optional<String> applicationServer;
```

Comment:
How does this compare to `applicationId` above?

Reply:
`applicationId` is used to identify different consumers of a Kafka topic. This lets multiple applications (or, more generically, consumer groups in Kafka lingo) consume one topic, ensuring that each message is consumed by each consumer group. So if you had two different Kafka Streams applications processing the same topic, they'd have to use different application ids.

`applicationServer` is used to identify individual nodes of one clustered Kafka Streams application. In this case each instance processes a subset of the partitions of the source topic(s). Kafka Streams provides the metadata APIs for finding out which instance of the clustered app is in charge of which messages (by means of assigning message keys to partitions), so that a consumer can identify the right node in a cluster for getting the state for a given key.

Reply:
Ah okay.  I didn't really get that from the property names or descriptions.

Reply:
Gotcha; to be fair I think it'll be apparent to Kafka Streams users. I'll also make the mapping apparent in the docs.